packages/account/src/Components/account-limits/account-limits.tsx

```diff
@@ -84,13 +83,8 @@ const AccountLimits = observer(
 
         if (is_virtual) {
             return (
-                <div
-                    data-testid='dt_account_demo_message_wrapper'
-                    className={classNames('account__demo-message-wrapper', {
-                        'account__demo-message-wrapper-dashboard': is_appstore,
-                    })}
-                >
-                    <DemoMessage has_demo_icon={is_appstore} has_button={is_appstore} />
+                <div data-testid='dt_account_demo_message_wrapper' className={'account__demo-message-wrapper'}>
```

Comment:
```suggestion
                <div data-testid='dt_account_demo_message_wrapper' className='account__demo-message-wrapper'>
```

Reply:
Fixed. 
Thanks @matin-deriv for highlighting.